### PR TITLE
Consistent use of breadcrumbs component

### PR DESCRIPTION
**Pull request by richtier**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] Add a printscreen to the PR

image

### CODE REVIEW

File: article/templates/article/article_detail_content.html [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {% load parse_date from article_tags %}
-{% load add_export_elements_classes from directory_components %}
+{% load add_export_elements_classes breadcrumbs from directory_components %}
 
 {% block breadcrumbs %}
-  {% include 'core/includes/breadcrumb.html' with breadcrumbs=breadcrumbs %}
+  {% with current_page=breadcrumbs|last %}
```
Comment (eadpearce) [inline article/templates/article/article_detail_content.html]: for all CMS pages we can use `article/components/cms_breadcrumbs.html` that picks up the page's `tree_based_breadcrumbs` list

File: article/templates/article/international_news_detail_content.html [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-      <li>
-        <a href="{% url 'international-eu-exit-news-list' %}">Brexit news</a>
-      </li>
-      <li aria-current="page">{{ page.article_title }}</li>
-    </ol>
-  </nav>
-</div>
+  <div class="container">
+    {% breadcrumbs page.article_title %}
+      <a href="{% url 'landing-page-international' %}">great.gov.uk</a>
```
Comment (eadpearce) [inline article/templates/article/international_news_detail_content.html]: if this is going to the international homepage the label should be 'Great.gov.uk international' as it is in great-international-ui

File: article/templates/article/domestic_news_detail_content.html [hunk 3]
```diff
@@ -1,21 +1,14 @@
...
-      </li>
-      <li>
-        <a href="{% url 'eu-exit-news-list' %}">Brexit news</a>
-      </li>
-      <li aria-current="page">{{ page.article_title }}</li>
-    </ol>
-  </div>
+  {% breadcrumbs page.article_title %}
+    <a href="{% url 'landing-page' %}">great.gov.uk</a>
+    <a href="{% url 'eu-exit-news-list' %}">Brexit news</a>
```
Comment (eadpearce) [inline article/templates/article/domestic_news_detail_content.html]: shouldn't we also rename the url to be consistent? may be out of scope of this PR

File: article/templates/article/article_detail_content.html [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+    {% breadcrumbs current_page.title %}
+      <a href="{% url 'landing-page' %}">great.gov.uk</a>
```
Comment (eadpearce) [inline article/templates/article/article_detail_content.html]: afaik when it's at the start of a sentence we should be capitalising Great.gov.uk but to be sure, ask someone in content

File: casestudy/templates/casestudy/base.html [hunk 5]
```diff
@@ -13,13 +13,7 @@
...
 {% block content %}
 <div class="container">
-    <div class="breadcrumbs body-text">
-      <ol>
-        <li><a href="{% url 'landing-page' %}">Export</a></li>
-        <li aria-current="page">{{ casestudy.title }}</li>
-      </ol>
-    </div>
-
+    {% breadcrumbs casestudy.title %}<a href="{% url 'landing-page' %}">Export</a>{% endbreadcrumbs %}
```
Comment (eadpearce) [inline casestudy/templates/casestudy/base.html]: the homepage label on this page is super old. we should update it to 'Great.gov.uk' like all the others

File: article/templates/article/country_guide.html [hunk 6]
```diff
@@ -15,7 +15,14 @@
 
 <section id="breadcrumb-section">
   {% block breadcrumbs %}
```
Comment (jessgilbert) [inline article/templates/article/country_guide.html]: Should there be `{% block breadcrumbs` as well as `{% breadcrumbs ...`

Comment (jessgilbert) [reply article/templates/article/country_guide.html]: Oh never mind I see, breadcrumbs is the block from directory components and just `breadcrumbs` is the bit of the block you're adding to?

File: article/templates/article/international_news_detail_content.html (hunk 2 above)
Comment (jessgilbert) [inline article/templates/article/international_news_detail_content.html]: Should this not have a capital `G` when in breadcrumbs? `Great.gov.uk`

File: article/templates/article/marketing_article_detail_content.html [hunk 7]
```diff
@@ -1,8 +1,15 @@
...
 
 {% block breadcrumbs %}
-  {% include 'article/components/cms_breadcrumbs.html' %}
+  {% with current_page=page.tree_based_breadcrumbs|last %}
+    {% breadcrumb current_page.title %}
+      <a href="{% url 'landing-page' %}">great.gov.uk</a>
+      {% for crumb in page.tree_based_breadcrumbs|slice:':-1' %}
+        <a href="{{ crumb.url }}">{{ crumb.title }}</a>
+      {% endfor %}
+    {% endbreadcrumb %}
```
Comment (jessgilbert) [inline article/templates/article/marketing_article_detail_content.html]: endbreadcrumbs - missing s

File: article/templates/article/domestic_news_detail_content.html (hunk 3 above)
Comment (richtier) [reply article/templates/article/domestic_news_detail_content.html]: good idea. will do in follow up PR.109 files changed already :)

File: article/templates/article/marketing_article_detail_content.html (hunk 7 above)
Comment (richtier) [reply article/templates/article/marketing_article_detail_content.html]: well spotted :) thanks

File: article/templates/article/domestic_news_detail_content.html [hunk 8]
```diff
@@ -1,23 +1,16 @@
...
-      <li>
-        <a href="{% url 'landing-page' %}">great.gov.uk</a>
-      </li>
-      <li>
-        <a href="{% url 'eu-exit-news-list' %}">Brexit news</a>
-      </li>
-      <li aria-current="page">{{ page.article_title }}</li>
-    </ol>
-  </div>
+  {% block breadcrumbs %}
```
Comment (eadpearce) [inline article/templates/article/domestic_news_detail_content.html]: this could use the cms breadcrumbs as well

File: article/templates/article/domestic_news_list.html [hunk 9]
```diff
@@ -18,20 +18,12 @@ <h1 class="heading-hero-generic-compact" id="hero-heading">{{ page.landing_page_
...
-  <nav class="breadcrumbs">
-    <ol>
-      <li>
-        <a href="{% url 'landing-page' %}">great.gov.uk</a>
-      </li>
-      <li aria-current="page">{{ page.landing_page_title }}</li>
-    </ol>
-  </nav>
-</div>
+{% block breadcrumbs %}
```
Comment (eadpearce) [inline article/templates/article/domestic_news_list.html]: same here

File: article/templates/article/international_news_detail_content.html [hunk 10]
```diff
@@ -1,19 +1,12 @@
...
+
 {% block breadcrumbs %}
-<div class="container">
-  <nav class="breadcrumbs" aria-label="breadcrumb" role="navigation">
-    <ol>
-      <li>
-        <a href="{% url 'landing-page-international' %}">great.gov.uk</a>
-      </li>
-      <li>
+    {% breadcrumbs page.article_title %}
```
Comment (eadpearce) [inline article/templates/article/international_news_detail_content.html]: actually i think these international news pages are no longer accessible because of the /international url going to great-international-ui now. but that can be another PR

File: contact/templates/contact/guidance-exporting-to-the-uk.html [hunk 11]
```diff
@@ -10,7 +10,13 @@
 {% endblock %}
 
 {% block content %}
-    {% include 'contact/breadcrumbs.html' %}
+    <div class="container">
+        {% breadcrumbs 'Exporting to the UK' %}
+            <a href="{% url 'landing-page' %}">Home</a>
```
Comment (eadpearce) [inline contact/templates/contact/guidance-exporting-to-the-uk.html]: does this want to be "great.gov.uk" rather than "Home"?

File: contact/templates/contact/guidance.html [hunk 12]
```diff
@@ -10,7 +10,12 @@
 {% endblock %}
 
 {% block content %}
-    {% include 'contact/breadcrumbs.html' %}
+    <div class="container">
+        {% breadcrumbs 'Guidance' %}
+            <a href="{% url 'landing-page' %}">Home</a>
```
Comment (eadpearce) [inline contact/templates/contact/guidance.html]: same here

File: core/templates/core/breadcrumbs_cms.html [hunk 13]
```diff
@@ -0,0 +1,15 @@
+{% load breadcrumbs from directory_components %}
+
+{% with current_page=page.tree_based_breadcrumbs|last %}
+
+{% breadcrumbs current_page.title %}
+    <a href="/">great.gov.uk</a>
```
Comment (eadpearce) [inline core/templates/core/breadcrumbs_cms.html]: this can actually be left out here because the homepage is in the cms and will be serialized in the tree based breadcrumbs

File: core/templates/core/performance_dashboard.html [hunk 14]
```diff
@@ -16,21 +16,16 @@
...
-        <li aria-current="page">Performance dashboard</li>
-      {% else %}
-      <li>
-        <a href="{% url 'performance-dashboard' %}">Performance dashboard</a>
-      </li>
-      <li aria-current="page">{{ page.heading }}</li>
-      {% endif %}
-    </ol>
-  </nav>
+  {% if page.landing_dashboard %}
```
Comment (eadpearce) [inline core/templates/core/performance_dashboard.html]: performance dashboard pages were updated to follow tree based structure in the cms so you can use the cms breadcrumbs here

File: core/templates/core/performance_dashboard_notes.html [hunk 15]
```diff
@@ -12,16 +14,9 @@
...
-      <li>
-        <a href="/">great.gov.uk</a>
-      </li>
-      <li>
-        <a href="{% url 'performance-dashboard' %}">Performance dashboard</a>
-      </li>
-      <li aria-current="page">{{ page.title }}</li>
-    </ol>
-  </div>
+  {% breadcrumbs page.title %}
```
Comment (eadpearce) [inline core/templates/core/performance_dashboard_notes.html]: same here

File: marketing/templates/marketing/join-success.html [hunk 16]
```diff
@@ -10,9 +10,9 @@
 
 {% block content %}
     <div class="container">
-		{% block breadcrumbs %}
-			{% include 'marketing/breadcrumbs.html' with current_page_label='Marketing Template page' %}
-		{% endblock %}
+        {% breadcrumbs 'Sent' %}
+            <a href="{% url 'landing-page' %}">great.gov.uk</a>
```
Comment (eadpearce) [inline marketing/templates/marketing/join-success.html]: shouldn't this have another link in the middle? either to the form or to the page the user came from

File: contact/templates/contact/guidance-exporting-to-the-uk.html (hunk 11 above)
Comment (richtier) [reply contact/templates/contact/guidance-exporting-to-the-uk.html]: for consistency. sure
